packages/frontend/src/components/ChartConfigPanel/index.tsx

```diff
@@ -7,10 +7,7 @@ import ChartConfigTabs from './ChartConfigTabs';
 
 export const ChartConfigPanel: React.FC = () => {
     const { chartType, plotData } = useVisualizationContext();
-    const disabled =
```

Comment:
I think we basically want a different configuration panel for big numbers and cartesian rather than reusing the cartesian configuration, for example, the tabs aren't needed